examples/pytorch/image-classification/run_image_classification_no_trainer.py

```diff
@@ -386,10 +399,11 @@ def collate_fn(examples):
 
     # We need to initialize the trackers we use, and also store our configuration
     if args.with_tracking:
-        experiment_config = vars(args)
-        # TensorBoard cannot log Enums, need the raw value
-        experiment_config["lr_scheduler_type"] = experiment_config["lr_scheduler_type"].value
-        accelerator.init_trackers("image_classification_no_trainer", experiment_config)
+        if accelerator.is_main_process:
+            experiment_config = vars(args)
+            # TensorBoard cannot log Enums, need the raw value
+            experiment_config["lr_scheduler_type"] = experiment_config["lr_scheduler_type"].value
+            accelerator.init_trackers("image_classification_no_trainer", experiment_config)
```

Comment:
Would it make more sense to have init_trackers only work on process 0 in this case? Rather than having the user do this

Reply:
Oh definitely!

Reply:
Yes, that would be better way of handling this because `accelerator.log` only logs only main process.

However, there are scenarios wherein one would want to log to multiple processes. For example, while experimenting with FSDP, I wanted to log GPU memory consumed by each GPU process. The logger would write the GPU memory stats to separate text files with one for each process.

Reply:
Let's include a small comment for why we want to do that in this case then, and think about expanding the tracking docs to discuss this

Reply:
Added comment.